### DISCUSSION TITLE
IOS Safari component rendering issues

### DISCUSSION THREAD

**Discussion by Colourcloud** [vercel/next.js#54091] [Help]
### Summary

Ever since I moved to nextjs 13 I’ve noticed on mobile, specifically IOS safari, when you scroll back to components that should have already rendered in, they are blank and have to load again (flickering effect)

This doesn't happen on desktop or Andriod devices that I tested it on

It’s also using tailwindcss

Can't seem to find any discussion of this issue so thought I'd ask here.

Heres a video showcasing the issue:
https://youtube.com/shorts/3WKbJDw38fs?feature=share

### Additional information

```js
Tested on:
IOS: 16.5.1
Iphone: 12, 12pro
```

### Example

bf-new.vercel.app

**Comment by Colourcloud** [comment]
Seems to be when its on lower power mode, and only affecting the areas where there are gradients (background and text)

Tested on an Iphone 11 and can't produce the same problem. Iphone 13 also works fine.

**Reply by HasanMothaffar** [reply]
I was facing the same issue, only on IOS devices (android works fine). There was a page where components flicker each time the user scrolls. It turns out that the problem was caused by a div that has `filter: drop-shadow`. All things were fine after removing this shadow.
